starfish/image/_stack.py

```diff
@@ -287,7 +288,12 @@ def show_stack(
         data, remaining_inds = self.get_slice(indices)
 
         # identify the dimensionality of data with all dimensions other than x, y linearized
-        n = numpy.dot(*data.shape[:-2])
+        if len(data.shape) >= 3:
+            n = numpy.product(data.shape[:-2])
+        else:
+            raise ValueError(
+                f'a stack with dimensionality >= 3 is required, the provided indexer produced a '
```

Comment:
string quote style.

Reply:
Yep, these problems are all over the place, will fix all at once in #321